ep/build-velox/src/get_velox.sh

```diff
@@ -17,7 +17,7 @@
 set -exu
 
 VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2024_01_19
+VELOX_BRANCH=2024-01-22
```

Comment:
Minor, we and the robot mixed use of "yyyy_MM_dd" and "yyyy-MM-dd" for the branch name. cc @rui-mo

Reply:
I find we are using `yyyy_mm_dd` for manually rebase but `yyyy-mm-dd` for auto rebase. Shall we align them? @zhouyuan Replace this PR with https://github.com/oap-project/gluten/pull/4486 for this time.

Reply:
Ack. Just modified the script from bot.

thanks, -yuan